### PR TITLE
Ceds 3513: Consignment references allow spaces on LRN

### PR DESCRIPTION
**Pull request by darrylmurphy**

### CODE REVIEW

File: test/forms/declaration/ConsignmentReferencesSpec.scala [hunk 1]
```diff
@@ -339,6 +353,12 @@ object ConsignmentReferencesSpec {
     data.asInstanceOf[JsObject].deepMerge(JsObject(Map("eidrDateStamp" -> JsString(eidrDateStamp))))
 
   val correctConsignmentReferencesJSON: JsValue = JsObject(Map("ducr" -> JsObject(Map("ducr" -> JsString(ducr))), "lrn" -> JsString(lrn)))
+  val correctConsignmentReferencesLrnWithUntrimmmedSpacesJSON: JsValue = JsObject(
+    Map("ducr" -> JsObject(Map("ducr" -> JsString(ducr))), "lrn" -> JsString(s" $lrn "))
+  )
+  val correctConsignmentReferencesLrnWithSpacesJSON: JsValue = JsObject(
+    Map("ducr" -> JsObject(Map("ducr" -> JsString(ducr))), "lrn" -> JsString(lrnWithSpaces))
```
Comment (udinobi) [inline test/forms/declaration/ConsignmentReferencesSpec.scala:360]: Same here

File: test/forms/declaration/ConsignmentReferencesSpec.scala [hunk 2]
```diff
@@ -339,6 +353,12 @@ object ConsignmentReferencesSpec {
     data.asInstanceOf[JsObject].deepMerge(JsObject(Map("eidrDateStamp" -> JsString(eidrDateStamp))))
 
   val correctConsignmentReferencesJSON: JsValue = JsObject(Map("ducr" -> JsObject(Map("ducr" -> JsString(ducr))), "lrn" -> JsString(lrn)))
+  val correctConsignmentReferencesLrnWithUntrimmmedSpacesJSON: JsValue = JsObject(
+    Map("ducr" -> JsObject(Map("ducr" -> JsString(ducr))), "lrn" -> JsString(s" $lrn "))
```
[the remark this answers is not in the document]
Comment (darrylmurphy) [reply test/forms/declaration/ConsignmentReferencesSpec.scala:357]: For consistency I used the same format as the existing val above, shall I change both or leave?

[the remark this answers is not in the document]
Comment (darrylmurphy) [reply test/forms/declaration/ConsignmentReferencesSpec.scala:357]: Yeah of course that works for me too. I didn't want to change/improve the code I'd added and left behind that existing code. I'm happy to make those changes now

[the remark this answers is not in the document]
Comment (darrylmurphy) [reply test/forms/declaration/ConsignmentReferencesSpec.scala:357]: Yeah of course that works for me too. I didn't want to change/improve the code I'd added and left behind that existing code. I'm happy to make those changes now

**Pull request merged**